### PR TITLE
🧹 replace ioutil.ReadFile with os.ReadFile

### PR DESCRIPTION
**Pull request by chris-rock**
replaces the deprecated ioutil.ReadFile method with os.ReadFile

Signed-off-by: Christoph Hartmann <chris@lollyrock.com>

### CODE REVIEW

File: resources/runtime.go [hunk 1]
```diff
@@ -64,7 +64,6 @@ func (c *Cache) Delete(key string) { c.Map.Delete(key) }
 type UpstreamConfig struct {
 	AssetMrn    string
 	SpaceMrn    string
-	Collector   string
```
Comment (imilchev) [inline resources/runtime.go]: is this change intended since it doesn't really fit into the context of this PR? Just making sure it doesn't disappear by accident 😆

Comment (chris-rock) [reply resources/runtime.go]: That is a change for another PR, will remove it from this here
